libbeat/_meta/fields.ecs.yml

```diff
@@ -1,2248 +1,2130 @@
-            In the OSI Model this would be the Network Layer. ipv4, ipv6, ipsec, pim, etc
-
-            The field value must be normalized to lowercase for querying. See
-            "Lowercase Capitalization" in the "Implementing ECS" section.
-          example: ipv4
-
-        - name: iana_number
-          level: extended
-          type: keyword
-          description: >
-              IANA Protocol Number (https://www.iana.org/assignments/protocol-numbers/protocol-numbers.xhtml).
-              Standardized list of protocols. This aligns well with NetFlow and
-              sFlow related logs which use the IANA Protocol Number.
-          example: 6
-
-        - name: transport
-          level: core
-          type: keyword
-          description: >
-            Same as network.iana_number, but instead using the Keyword name of the
-            transport layer (udp, tcp, ipv6-icmp, etc.)
-
-            The field value must be normalized to lowercase for querying. See
-            "Lowercase Capitalization" in the "Implementing ECS"  section.
-          example: tcp
-
-        - name: application
-          level: extended
-          type: keyword
-          description: >
-            A name given to an application. This can be arbitrarily assigned for
-            things like microservices, but also apply to things like skype, icq,
-            facebook, twitter. This would be used in situations where the vendor
-            or service can be decoded such as from the source/dest IP owners,
-            ports, or wire format.
-
-            The field value must be normalized to lowercase for querying. See
-            "Lowercase Capitalization" in the "Implementing ECS" section.
-          example: aim
-
-        - name: protocol
-          level: core
-          type: keyword
-          description: >
-            L7 Network protocol name. ex. http, lumberjack, transport protocol.
-
-            The field value must be normalized to lowercase for querying. See
-            "Lowercase Capitalization" in the "Implementing ECS" section.
-          example: http
-
-        - name: direction
-          level: core
-          type: keyword
-          description: >
-            Direction of the network traffic.
-
-            Recommended values are:
-              * inbound
-              * outbound
-              * internal
-              * external
-              * unknown
-
-            When mapping events from a host-based monitoring context, populate this
-            field from the host's point of view.
-
-            When mapping events from a network or perimeter-based monitoring context,
-            populate this field from the point of view of your network perimeter.
-          example: inbound
-
-        - name: forwarded_ip
-          level: core
-          type: ip
-          description: >
-            Host IP address when the source IP address is the proxy.
-          example: 192.1.1.2
-
-        - name
```

Comment:
I'm moving this to `fields.common.yml`. Hopefully this is the only one like this

Reply:
All tests now passing. Seems like it was.